docs/04-documentation.md

```diff
@@ -0,0 +1,36 @@
+---
+title: How to update documentation on ECL
+label: Edit documentation
+---
+
+This page explains how to update components documentation directly in GitHub
```

Comment:
`update documentation directly on GitHub` is a bit more concise.
Please add a fullstop in the end of the sentence.